.github/workflows/awesome_workflow.yml

```diff
@@ -23,10 +23,9 @@ jobs:
           git config --global user.name github-actions[bot]
           git config --global user.email 'github-actions@users.noreply.github.com'
       - name: Filename Formatter
-        run: |
-          wget https://raw.githubusercontent.com/TheAlgorithms/scripts/main/filename_formatter.sh
-          chmod +x filename_formatter.sh
-          ./filename_formatter.sh . .c,.h
+        uses: TheAlgorithms/scripts/formatter@main
```

Comment:
`TheAlgorithms/scripts` is the repo and `formatter` is the folder which contains the `action.yml` file that describes the action. The `@main` is necessary because its done through commit sha's or through the branch refs (in this case `main` is easier than using the commit hash `d6eba756e1ba7e43a5964f16b76a817c0cd54e65` so I used that)